modules/gdcv-for-baremetal/DEMO.md

```diff
@@ -0,0 +1,85 @@
+# Demo Install Steps
+
+- Install jq if you don't have it already
+
+https://jqlang.github.io/jq/download/
+
+- Install git if you don't have it already
+
+https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+
+```sh
+git clone https://github.com/cprivitere/terraform-gcpartner-anthos-baremetal
```

Comment:
Note for the future: once https://github.com/GCPartner/terraform-gcpartner-anthos-baremetal/pull/21 is merged, we should update this URL to point to the original repo instead of a personal fork